### PR TITLE
Introduce WPSTARTER_PATH in wp-config template

### PR DESCRIPTION
**Pull request by Giuseppe-Mazzapica**
See #30

### CODE REVIEW

**Comment by franz-josef-kaiser** [comment]
Absolutely the way I'd go. LGTM

File: wpstarter/templates/wp-config.example [hunk 1]
```diff
@@ -16,6 +16,11 @@
 require_once realpath({{{VENDOR_PATH}}}.'/autoload.php');
 
 /**
+ * A reference to `.env` folder path.
+ */
+define('WPSTARTER_PATH', {{{ENV_REL_PATH}}});
```
Comment (dnaber-de) [inline wpstarter/templates/wp-config.example]: Where does `ENV_REL_PATH` come from?

Comment (Giuseppe-Mazzapica) [inline wpstarter/templates/wp-config.example]: @dnaber-de All the variables used inide the `wp-config.example` template are generated during installation in the wp-config "step".

See https://github.com/wecodemore/wpstarter/blob/master/wpstarter/src/Setup/Steps/WPConfigStep.php#L99-L111

Comment (dnaber-de) [inline wpstarter/templates/wp-config.example]: Okay, thanks.

**Pull request merged**